apps/client/src/app/debug/chat.tsx

```diff
@@ -41,6 +41,11 @@ const ChatPage = ({
     }
   };
 
+  const sendStart = async (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault();
```

Comment:
No, I cannot find the solution to prevent react to send request 2 times in dev mode. Hence, I used variable 'rooms' to keep track current player each time they join or leave lobby. The 'preventDefault' function is used to just prevent the page from refresh after client click submit button.